package.json

```diff
@@ -11,7 +11,9 @@
     "@types/react": "^18.2.28",
     "@types/react-dom": "^18.2.13",
     "axios": "^1.6.2",
+    "bootstrap": "^5.3.2",
     "react": "^18.2.0",
+    "react-bootstrap": "^2.9.1",
```

Comment:
Bootstrap jest potrzebny aby używać Modala - to ten komunikat przy potwierdzaniu usuwania.

![image](https://github.com/pawelNu/compstore-ui/assets/93542936/d1a7ea86-4459-4cd4-89ec-4c44a6235661)

Może jak będziemy większymi kozakami to zamiast używać komponentów napisanych niemal w całości w HTML będziemy używać tych Reactowych z zależności Bootstrap. To jest to samo tylko, bardziej Typescript. Wspomnę jeszcze o tym przy Modalu.

Reply:
Tak jak pisałeś w czacie - jestem za korzystaniem z gotowych elementów